tests/test_rtdc_feat_anc_plugin.py

```diff
@@ -905,6 +964,186 @@ def test_pf_wrong_length_2():
             ds[pf.feature_name]
 
 
+@pytest.mark.filterwarnings(
+    "ignore::dclab.rtdc_dataset.config.WrongConfigurationTypeWarning")
+def test_pf_store_scalar_plugin_data_with_shape_info():
+    """Test storage of scalar plugin feature attributes"""
+    h5path = retrieve_data("fmt-hdf5_fl_2018.zip")
+    with dclab.new_dataset(h5path) as ds:
+        circ_per_area = ds["circ"] / ds["area_um"]
+    info = example_plugin_info_single_feature()
+    info["feature shapes"] = [circ_per_area.shape]
+    PlugInFeature("circ_per_area", info)
+    with RTDCWriter(h5path) as hw:
+        hw.store_feature("circ_per_area", circ_per_area)
+    with h5py.File(h5path, mode="r") as h5:
+        events = h5["events"]
+        assert "circ_per_area" in events.keys()
+        feat = h5["events"]['circ_per_area']
+        assert feat.ndim == 1
+        assert not isinstance(feat[0], np.ndarray)
+        assert not feat.attrs.keys()
+
+
```

Comment:
Have you done this?

Reply:
You can find the screenshot here (https://github.com/ZELLMECHANIK-DRESDEN/dclab/pull/172#issuecomment-1081930071)

Reply:
Aha, cool. Maybe actually place it in the original issue so that general users can see it! And a screenshot of the actual image open in HDFView too!

Reply:
Yes, you can find here (https://github.com/ZELLMECHANIK-DRESDEN/dclab/issues/171#issue-1178295751)